src/sinonSandbox.js

```diff
@@ -0,0 +1,25 @@
+export function isCreateSandbox(value, t) {
+    return t.isCallExpression(value.expression.right)
+        && value.expression.right.callee.property.name === 'createSandbox'
+}
+
+export function isRestoreSandbox(value, t) {
+  return t.isMemberExpression(value.expression.callee)
+      && t.isMemberExpression(value.expression.callee.object)
+      && value.expression.callee.object.property.name === 'ss'
```

Comment:
I see a couple instances where instead of `ss` it is `sinonbox`, could we cover that too?

https://github.com/okta/okta-ui/blob/master/packages/admin/appinstance/test/spec/assignments/views/components/PeopleAssignmentActionView_spec.js#L28

Reply:
Yeah, I can make it dynamic as what we did on the import. I will work on that after other things done.